doc/modules/ROOT/content-nav.adoc

```diff
@@ -18,4 +18,7 @@ include::partial$tutorial-list.adoc[]
 
 * xref:known-limitations.adoc[]
 
+* Appendix
```

Comment:
It is temporary. Eventually this will become top of mind and receive a slot very early in the documentation. We were indecisive so we invented an Appendix. Happy to reconsider if you have suggestions!

Reply:
I think it can just be at the same level as the other nav items (so replacing the "Appendix"), but I understand if you want to keep it "aside" for now. I'll leave the decision to you.

Reply:
I'm opting for your suggestion then, and will just unnest it and remove `Appendix`.

Reply:
I moved it up above the Tutorials now.